starcheck/calc_ccd_temps.py

```diff
@@ -404,7 +410,7 @@ def get_telem_values(tstart, msids, days=7, name_map={}):
     start = DateTime(tstart - days * 86400).date
     stop = DateTime(tstart).date
     logger.info('Fetching telemetry between %s and %s' % (start, stop))
-    msidset = fetch.MSIDset(msids, start, stop, stat='5min')
+    msidset = fetch.MSIDset(msids, start, stop)
```

Comment:
I think it is still fine to interpolate in 5 minute steps below, but if we are just fetching the one msid the interpolation doesn't seem necessary anyway.

Reply:
Is the interpolation necessary or can it be cut?

Reply:
I don't know.  I haven't dug into the code to see how it gets used, so your call.